dynamodb-enhanced/runtime/src/main/java/io/quarkus/amazon/dynamodb/enhanced/runtime/BuildTimeConfig.java

```diff
@@ -0,0 +1,13 @@
+package io.quarkus.amazon.dynamodb.enhanced.runtime;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "dynamodb-enhanced", phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
+public class BuildTimeConfig {
+
+    /** Apply patch for crashing tests. Reduces performance in JVM mode. */
+    @ConfigItem(defaultValue = "true")
+    public boolean jvmTransformation;
```

Comment:
Could you make me a summary of what this is supposed to fix?

Reply:
This is only a flag which you can set if you do not want to use the fix when in LaunchMode.DEVELOPMENT and LaunchMode.TEST.

Reply:
I agree we should just remove this flag.